api/src/uvisor-input.S

```diff
@@ -77,7 +77,7 @@ uvisor_config:
     /* Address of __uvisor_box_context */
     .long __uvisor_box_context
 
-    /* Main heap for box 0 */
+    /* Public heap for box 0 */
```

Comment:
Consider saying "Public heap" only, dropping "for box 0".

One may think, "Why mention box 0? Are there non-public heaps for box 0?"